tests/phpt/error_handler_captures_exception.phpt

```diff
@@ -41,6 +41,3 @@ throw new \Exception('foo bar');
 --EXPECTF--
 Exception listener called
 Custom exception handler called
-Fatal error: Uncaught Exception: foo bar in %s:%d
-Stack trace:
-%a
```

Comment:
I changed this expectation because it's what is happening if I remove completely Sentry from the test.